components/alert/alert.css

```diff
@@ -0,0 +1,53 @@
+.alert {
+  width: 100%;
+  padding: 1rem 1.5rem;
+  margin: 1rem;
+  background-color: var(--color-default);
+  border-radius: 25px;
```

Comment:
You must use rems instead of px and pass it as a variable if this border radius value will be used frequently.

Reply:
Hey, thanks for reviewing. 
So, I did try rems but there wasn't any change so I thought I'll stick with px to give it a fixed border-radius to facilitate the same border on smaller screens too. Would love to discuss your POV.